lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,9 +27,39 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instances(instances)
+          requires :identity, :zone
+
+          service.add_instance_group_instances(
+            identity, zone_name, format_instance_list(instances)
+          )
+        end
+
+        def remove_instances(instances)
+          requires :identity, :zone
+
+          service.remove_instance_group_instances(
+            identity, zone_name, format_instance_list(instances)
+          )
+        end
+
+        def list_instances
```

Comment:
@Temikus @icco Agreed. You might be able to make it easier using the `Fog` DSL for associations.